File: 217-contains-duplicate/217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        hashSet = set()
        for num in nums:
                if num in hashSet:
                        return True
                hashSet.add(num)
        return False
        

File: 217-contains-duplicate/217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    hashSet = new Set()
    for (let i=0; i<nums.length; i++){
        if (hashSet.has(nums[i])){
            return true
        }
        hashSet.add(nums[i])
    }
    return false
};